src/views/Header/Header.js

```diff
@@ -641,17 +641,16 @@ function warnIfHeaderStyleDefined(value, styleProp) {
   }
 }
 
-let platformContainerStyles;
-if (Platform.OS === 'ios') {
-  platformContainerStyles = {
+const platformContainerStyles = Platform.select({
+  android: {
+    elevation: 4,
+  },
+  ios: {
     borderBottomWidth: StyleSheet.hairlineWidth,
     borderBottomColor: '#A7A7AA',
-  };
-} else {
-  platformContainerStyles = {
-    elevation: 4,
-  };
-}
+  },
+  default: {},
```

Comment:
Maybe we also want to add shadows for web?

Reply:
ya this sounds good. let's do it in follow up pr